rcl_yaml_param_parser/src/parser.c

```diff
@@ -967,7 +967,10 @@ static rcl_ret_t parse_value(
             return RCL_RET_BAD_ALLOC;
           }
         } else {
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wmaybe-uninitialized"
```

Comment:
This needs to be conditionally done, as it will not work on Windows. For example: https://github.com/ros2/rviz/blob/f594d9f00ecf87f75a0a3909f538bd463f7b7375/rviz_rendering/include/rviz_rendering/objects/point_cloud.hpp#L40-L63